project_0b/exercise_3.c

```diff
@@ -0,0 +1,45 @@
+/*
+Project 0b
+*/
+
+/*
+Exercise 3 a - According to Malthus Law: x(n+1) = µ x(n), defined within [0, 1]
+for both µ and x.
+
+Write a program that reads user input for x0 and µ, and shows the first 20
+iterations of the equation.
+
+Exercise 3 b - Add an option for the user to define the total number of
+iterations.
+*/
+
+#include <math.h>
+#include <stdio.h>
+
+int malthus_law_calculator(double x_initial, double growth_rate, int interations) {
+  double instant_growth = x_initial;
+  printf("\nx0 = %.2lf | µ = %.2lf\n", x_initial, growth_rate);
+  for (int counter = 0; counter < interations; counter++) {
```

Comment:
nit: for a simple iteration counter, consider `i` instead of `counter`. It's obvious to everyone and reduces verbosity in a good way.